app/scripts/controllers/preferences.js

```diff
@@ -92,8 +92,6 @@ export default class PreferencesController {
         redesignedConfirmationsEnabled: true,
         redesignedTransactionsEnabled: true,
         featureNotificationsEnabled: false,
-        showTokenAutodetectModal: null,
-        showNftAutodetectModal: null, // null because we want to show the modal only the first time
```

Comment:
Depending on which version this goes in, these may need to be removed from state in a migration.

If it goes in before `showNftAutodetectModal` reaches users, it won't need to be deleted since it won't have entered state yet.